### PR TITLE
Updated to Netstandard 2.0 and added VB.NET example project

### PR DESCRIPTION
**Pull request by TheConservativeCoder**
Hi,

I've updated fluid to netstandard 2.0 and have included an example project that mirrors your example except that it uses VB.NET instead.

### CODE REVIEW

File: Fluid.MvcSampleVB/Program.vb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Imports System
+Imports System.IO
+Imports Microsoft.AspNetCore.Hosting
+
+Module Program
+    Sub Main(args As String())
+        Dim host = New WebHostBuilder() _
```
Comment (hishamco) [inline Fluid.MvcSampleVB/Program.vb]: I don't think the line continuation is needed here

File: Fluid.MvcSampleVB/Startup.vb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+﻿Imports Microsoft.Extensions.DependencyInjection
```
Comment (hishamco) [inline Fluid.MvcSampleVB/Startup.vb]: Sort the namespaces

File: Fluid.MvcSampleVB/Startup.vb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Imports FluidMvcViewEngine
+Imports Microsoft.AspNetCore.Builder
+Imports Microsoft.AspNetCore.Hosting
+Imports Microsoft.Extensions.Logging
+
+Public Class Startup
+    Shared Sub New()
```
Comment (hishamco) [inline Fluid.MvcSampleVB/Startup.vb]: Why the constructor is `Shared`

File: Fluid.MvcSampleVB/Startup.vb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        TemplateContext.GlobalMemberAccessStrategy.Register(Of Person)()
+    End Sub
+
+    Public Sub ConfigureServices(services As IServiceCollection)
+        services.AddMvc() _
```
Comment (hishamco) [inline Fluid.MvcSampleVB/Startup.vb]: same as previous comment

File: Fluid.MvcSampleVB/Program.vb (hunk 1 above)
Comment (TheConservativeCoder) [reply Fluid.MvcSampleVB/Program.vb]: Do you mean that the newline format should be kept except without the `_`, or should it all be done all on one line?

Comment (hishamco) [reply Fluid.MvcSampleVB/Program.vb]: Nope, VB multi line continuation, in declaration case you can try this  
```vb
Call New WebHostBuilder().
            UseKestrel().
            UseContentRoot(Directory.GetCurrentDirectory()).
            UseIISIntegration().
            UseStartup(Of Startup)().
            Build().
            Run()
```

Comment (hishamco) [reply Fluid.MvcSampleVB/Program.vb]: Or you can use `With` statement if you do so

Comment (TheConservativeCoder) [reply Fluid.MvcSampleVB/Program.vb]: Sweet, Didn't know about the `.` on the end of a line for it to continue!

File: Fluid.MvcSampleVB/Program.vb [hunk 5]
```diff
@@ -4,7 +4,12 @@ Imports Microsoft.AspNetCore.Hosting
 
 Module Program
     Sub Main(args As String())
-        Dim host = New WebHostBuilder().UseKestrel().UseContentRoot(Directory.GetCurrentDirectory()).UseIISIntegration().UseStartup(Of Startup)().Build()
+        Dim host = New WebHostBuilder().
```
Comment (hishamco) [inline Fluid.MvcSampleVB/Program.vb]: Add missing `As`

**Comment by hishamco** [comment]
Thanks for your contribution @TheConservativeCoder, don't forget to squash your commits
@sebastienros hope it's fine for you

File: Fluid.MvcSampleVB/Program.vb [hunk 6]
```diff
@@ -4,7 +4,7 @@ Imports Microsoft.AspNetCore.Hosting
 
 Module Program
     Sub Main(args As String())
-        Dim host = New WebHostBuilder().
+        Dim host As IWebHost = New WebHostBuilder().
```
Comment (hishamco) [inline Fluid.MvcSampleVB/Program.vb]: It's enough to use
```vb
Dim host As New WebHostBuilder()
```

Comment (TheConservativeCoder) [reply Fluid.MvcSampleVB/Program.vb]: If it is done like that I would then be unable to call `host.Run()` as it belongs to `IWebHost`?

Comment (hishamco) [reply Fluid.MvcSampleVB/Program.vb]: I see, so that's why I surprised how `New WebHostBuilder().` it works with you without using `Call` 😄 

thanks

Comment (TheConservativeCoder) [reply Fluid.MvcSampleVB/Program.vb]: ahh I see you are calling it directly inline as one. I misread your example earlier 😊

Comment (hishamco) [reply Fluid.MvcSampleVB/Program.vb]: No problem even your latest suggestion was fine
thanks again

**Comment by sebastienros** [comment]
What is the point of targeting netstandard 2.0 when we can just require 1.6 ?

**Comment by TheConservativeCoder** [comment]
So that it is usable with VB.NET

**Comment by sebastienros** [comment]
Can we multi-target then?

**Comment by TheConservativeCoder** [comment]
Not sure what you mean? Are you saying VB.NET With ASP.NET Core 2 can reference Net Standard 1.6 or Net46?

**Comment by sebastienros** [comment]
just don't remove the netstandard1.6 when you add netstandard2.0, and use conditional <PropertyGroup and <ItemGroup>

**Comment by hishamco** [comment]
Multi targeting is a good option as @sebastienros said before

**Comment by TheConservativeCoder** [comment]
@sebastienros What you are saying is that I should enable multi targetting by setting it like this?
`<TargetFrameworks>netstandard1.6;net46;netstandard2.0</TargetFrameworks>`

I'm not majorly familar with MSBuild stuff but I also changed the Common.props file as it has `<TargetFrameworks>netstandard1.6</TargetFrameworks>` should this be left also?

**Comment by TheConservativeCoder** [comment]
Closing as replaced with #27.

**Comment by hishamco** [comment]
I'm not sure why you create a new PR, just add `netstandard2.0` to target frameworks

**Comment by TheConservativeCoder** [comment]
@hishamco I thought it would be easier as it would then be condensed down into small commits instead of lots of little ones. Appologies if this is not the right way, I'm not used to the etiquette of doing this with git and open (Used to using TFS instead).

**Comment by hishamco** [comment]
No problem I already such things on my early days of ASP.NET 5 😄 , nothing but you can add a commits, after that use `Squash` to make them single one

**Comment by sebastienros** [comment]
You don't even need to squash, we do it for you when we accept the PR (same in ASP.NET)

**Pull request closed without merging**